tests/pages/index.js

```diff
@@ -14,21 +14,22 @@ class PageObject {
 
 class CommentOnPage extends PageObject {
   body() {
-    return this.$('.comment-body').
-      text().
-      replace(/\s+/g, ' ').
-      replace(/^\s*/, '').
-      replace(/\s*$/, '');
+    return this.$('.comment-body')
+      .text()
+      .replace(/\s+/g, ' ')
+      .replace(/^\s*/, '')
+      .replace(/\s*$/, '');
   }
 
   parts() {
-    return this.$('.comment-parts li').
-      map((x,y) => self.jQuery(y).text()).
-      toArray().
-      map(x =>
-        x.replace(/\s+/g, ' ').
-        replace(/^\s*/, '').
-        replace(/\s*$/, '')
+    return this.$('.comment-parts li')
+      .map((x, y) => self.jQuery(y).text())
+      .toArray()
+      .map(x =>
+        x
+          .replace(/\s+/g, ' ')
```

Comment:
what has happened here?

seems strange to break like this when it's not necessary

Reply:
yeah this seems bad